### PR TITLE
Final AWS SDK v2 bits

### PR DESCRIPTION
**Pull request by Nirvikalpa108**
## What does this change?
This work completes moving Prism to AWS SDK v2 (apart from the remaining TODO). 

The reasons for this work are detailed in PR #95.

## TODO
We have kept bits of AWS SDK v1 in the codebase, because we need to do a separate piece of work on `DynamoConfiguration.scala`. Once that's complete we should be able to remove those final v1 dependencies.

### CODE REVIEW

File: app/utils/AWSCredentialProviders.scala [hunk 1]
```diff
@@ -2,23 +2,33 @@ package utils
 
 import com.amazonaws.auth.profile.{ProfileCredentialsProvider => ProfileCredentialsProviderV1}
 import com.amazonaws.auth.{AWSCredentialsProviderChain, DefaultAWSCredentialsProviderChain}
-import software.amazon.awssdk.auth.credentials.{AwsCredentialsProviderChain, DefaultCredentialsProvider, ProfileCredentialsProvider => ProfileCredentialsProviderV2}
+import software.amazon.awssdk.auth.credentials.{AwsCredentialsProviderChain, DefaultCredentialsProvider, ProfileCredentialsProvider => ProfileCredentialsProvider}
```
Comment (sihil) [inline app/utils/AWSCredentialProviders.scala:5]: You're no longer renaming this so you can remove the `=> Profile...` part.

File: build.sbt [hunk 2]
```diff
@@ -40,35 +40,28 @@ lazy val root = (project in file("."))
...
+      "software.amazon.awssdk" % "acm" % awsVersion,
+      "software.amazon.awssdk" % "ec2" % awsVersion,
+      "software.amazon.awssdk" % "autoscaling" % awsVersion,
+      "software.amazon.awssdk" % "elasticloadbalancing" % awsVersion,
+      "software.amazon.awssdk" % "route53" % awsVersion,
+      "software.amazon.awssdk" % "iam" % awsVersion,
+      "software.amazon.awssdk" % "kinesis" % awsVersion,
+      "com.beust" % "jcommander" % "1.75", // TODO: remove once security vulnerability introduced by aws sdk v2 fixed: https://snyk.io/vuln/maven:com.beust%3Ajcommanderbu
+      "com.amazonaws" % "aws-java-sdk-dynamodb" % awsVersionOne,
+      "com.amazonaws" % "aws-java-sdk-autoscaling" % awsVersionOne,
```
Comment (sihil) [inline build.sbt:56]: This dependency isn't used anywhere.

**Review by sihil — APPROVED** [review]
This looks good to me aside from a couple of minor comments. As long as it works 😄

**Comment by Nirvikalpa108** [comment]
This has been deployed to CODE successfully with no errors 🎉

**Review by sihil — APPROVED** [review]

**Pull request merged**